app/javascript/crayons/index.js

```diff
@@ -3,3 +3,4 @@ export * from '@crayons/ButtonGroup';
 export * from '@crayons/Dropdown';
 export * from '@crayons/formElements';
 export * from '@crayons/Modal';
+export * from '@crayons/Spinner';
```

Comment:
Allows us to import via the [webpack alias](https://webpack.js.org/configuration/resolve/#resolvealias) `@crayons`, e.g. `import { Spinner } from '@crayons';`